templates/password_reset.html

```diff
@@ -27,7 +27,7 @@ <h1 style="font-family: sans-serif; font-weight: normal; margin: 0 0 24px 0; tex
   <td border="0" cellpadding="0" cellspacing="0" height="100%" width="100%">
     <br/>
     <p class="secondary" style="font-family: sans-serif; font-weight: normal; margin: 0; text-align: center; color: #8A9BA8; font-size: 11px; line-height: 13px; width: 310px !important; word-wrap:break-word">
-      {{t "This is an automated email; if you received it in error, no action is required."}} {{{t "For more information, please visit <a %(supportLinkAttributes)s>Mozilla Support</a>"}}}
+      {{t "This is an automated email; if you did not authorize this action, then <a %(passwordChangeLinkAttributes)s>please change your password.</a>" }} {{{t "For more information, please visit <a %(supportLinkAttributes)s>Mozilla Support</a>"}}}
```

Comment:
Ahh, since this now has HTML, you'll have to replace `{{` with `{{{`or else the HTML will be escaped.